ksql-execution/src/main/java/io/confluent/ksql/services/SimpleKsqlClient.java

```diff
@@ -69,13 +69,19 @@ void makeAsyncHeartbeatRequest(
   RestResponse<ClusterStatusResponse> makeClusterStatusRequest(URI serverEndPoint);
 
   /**
-   * Send a request to remote Ksql server to inquire to inquire about which state stores the
-   * remote server maintains as an active and standby.
-   * @param serverEndPoint the remote destination.
+   * Send a request to remote Ksql server to inquire to inquire about which state stores the remote
```

Comment:
```suggestion
   * Send a request to remote Ksql server to inquire about which state stores the remote
```
super nit (not your code): typo.

Reply:
ack... also shouldn't it be "enquire" ? (Or maybe that's a US vs UK thing, but inquire seems wrong to me)

Reply:
Yup, US vs. UK (though [this blog](https://www.grammarly.com/blog/inquire-enquire/) suggests both versions are used in the UK) -- haha!